apps/etterlatte-saksbehandling-ui/client/src/components/person/saksliste.tsx

```diff
@@ -68,3 +73,25 @@ function endringStatusNavn(status: IBehandlingStatus) {
       return status
   }
 }
+
+function resultatTekst(behandling: IBehandlingsammendrag): string {
+  switch (behandling.behandlingType) {
+    case IBehandlingsType.FØRSTEGANGSBEHANDLING:
+      return resultatTekstFoerstegangsbehandling(behandling.vilkaarsvurderingutfall)
+    case IBehandlingsType.REVURDERING:
+    case IBehandlingsType.MANUELT_OPPHOER:
```

Comment:
```suggestion
    case IBehandlingsType.MANUELT_OPPHOER:
        return behandling.status === IBehandlingStatus.AVBRUTT ? "Avbrutt" : "Opphørt i systemet for behandling i PeSys" 
```

Whatever med revurdering

Reply:
evt. ha default på `behandling.status`

Reply:
Da står det: "opphørt i systemet for behandling i pesys" även når behandlingen kun er opprettet men ikke iverksatt?

Reply:
ah nevermind, kan wrappe lengre opp med `erFerdigBehandlet(..)`